src/main/java/duke/commands/ShowAllCommands.java

```diff
@@ -0,0 +1,58 @@
+package duke.commands;
+
+import duke.commands.Command;
+import duke.ui.Ui;
+
+/**
+ * Class of all the valid commands user can input
+ */
+public class ShowAllCommands extends Command<String> {
+    /**
+     * Enumeration of all valid commands.
+     */
+    enum Commands {LIST, TODO, EVENT, DEADLINE, BYE, MARK, UNMARK, DELETE, FIND}
```

Comment:
I like how you put the commands as enum which makes your following code neater :thumbsup: